catalog/dags/common/sql.py

```diff
@@ -38,6 +40,14 @@ def single_value(cursor):
         raise ValueError("Unable to extract expected row data from cursor") from e
 
 
+def fetch_all(cursor):
+    try:
+        rows = cursor.fetchall()
+        return [row[0] for row in rows]
+    except Exception as e:
+        raise ValueError("Unable to extract expected row data from cursor") from e
```

Comment:
If we're running `fetch_all` on the cursor, why do we also need to return `row[0]` for every result? 😮 Also just noting that Airflow has a [`fetch_all_handler`](https://github.com/apache/airflow/blob/278dd31228f73f926eff68124374d0db1f4d70b8/airflow/providers/common/sql/hooks/sql.py#L93) that we could use if that operation isn't necessary.

Reply:
Yes, I would've preferred to use the existing handler but the results are returned as tuples. Unpacking/accessing the first element within `transform_index_defs` felt much less clean to me.